src_api/__init__.py

```diff
@@ -14,8 +14,8 @@
 formatter = logging.Formatter("%(levelname)s:%(name)s:%(message)s")
 
 log_file = "app.log"
-if os.path.exists(log_file):
-    os.remove(log_file)
+#if os.path.exists(log_file):
+#    os.remove(log_file)

```

Comment:
should be uncommented.
the issue you experience is only related to windows file system access rights. 
It works fine on linux or deployed to Heroku.